ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewCommandHelper.java

```diff
@@ -26,10 +26,12 @@
 
   public static final int COMMAND_SCROLL_TO = 1;
   public static final int COMMAND_SCROLL_TO_END = 2;
+  public static final int COMMAND_SCROLL_BY = 3;
```

Comment:
Just for visibility; I've also changed this file: https://github.com/facebook/react-native/pull/15566/files#diff-0e688b19a3a45a0d17cd8d922cc0580aR29

If that makes its way into the code-base, there may be a small rebase required :+1: